assignment2.py

```diff
@@ -0,0 +1,155 @@
+class ListNode:
+    def __init__(self, value):
+        self.next = None
+        self.value = value
+        
```

Comment:
The spacing convention in Python is to have two new lines between top-level class and/or function definitions.